app-ios/Sources/TimetableFeature/TimetableGridCard.swift

```diff
@@ -22,17 +25,19 @@ public struct TimetableGridCard: View {
         } label: {
             VStack(alignment: .leading, spacing: 8) {
                 HStack(spacing: 4) {
-                    RoomTypeShape(roomType: .init(enTitle: timetableItem.room.name.enTitle))
+                    if cellCount == 1 {
+                        RoomTypeShape(roomType: .init(enTitle: timetableItem.room.name.enTitle))
                         .foregroundStyle(timetableItem.room.roomTheme.primaryColor)
+                    }
                     Text("\(timetableItem.startsTimeString) - \(timetableItem.endsTimeString)")
                         .textStyle(.labelMedium)
-                        .foregroundStyle(timetableItem.room.roomTheme.primaryColor)
+                        .foregroundStyle(cellCount>1 ? AssetColors.Surface.onSurfaceVariant.swiftUIColor : timetableItem.room.roomTheme.primaryColor)
```

Comment:
```suggestion
                        .foregroundStyle(cellCount > 1 ? AssetColors.Surface.onSurfaceVariant.swiftUIColor : timetableItem.room.roomTheme.primaryColor)
```
I want add spaces for any operators left and right😅